assets/src/edit-story/elements/shared/editPanMovable.js

```diff
@@ -28,20 +28,20 @@ import { useEffect, useRef } from '@wordpress/element';
  * Internal dependencies
  */
 import Movable from '../../components/movable';
-import { getFocalFromOffset } from './util';
+import { getFocalFromOffset } from './';
```

Comment:
I think it's better to reference the files inside the package directly and keep `index.js` for external dependers.

Reply:
There is currently just one `index.js` file. Will move the functions separately though.